mithril.js

```diff
@@ -1706,16 +1706,16 @@
 		var currentTarget = e.currentTarget || e.srcElement
 		var args
 
+		while (currentTarget && !/^a$/i.test(currentTarget.nodeName)) {
```

Comment:
You're right. An `a` used as a sub-resource identifier is a possibility and an edge case. As far as I know, all the other elements that provide an implicitly functional `href` that *don't* map to your behaviour are self-closing void elements (`link` and 1 or 2 others IIRC), so the check for `href` might actually be a future bug fix :)